frontend/js/src/utils/types.d.ts

```diff
@@ -56,7 +56,7 @@ declare type MBIDMapping = {
   release_mbid: string;
   artist_mbids: Array<string>;
   artists?: Array<MBIDMappingArtist>;
-  caa_id?: number | null;
+  caa_id?: number;
   caa_release_mbid?: string | null;
```

Comment:
And along with the comment I made above, perhaps `caa_release_mbid` too should  have the `| null` removed.